crates/turborepo-cache/src/http.rs

```diff
@@ -53,6 +63,7 @@ impl HTTPCache {
             client,
             signer_verifier,
             repo_root,
+            uploads: Arc::new(Mutex::new(HashMap::new())),
```

Comment:
This was originally just going to be a HashMap since the queries are safe to be cloned however I experienced race conditions where uploads were being added to the map _after_ the shutdown.